### PR TITLE
[NET-493] - Additional data fields to send to account management

### PR DESCRIPTION
**Pull request by gabrielseibel1**
## Describe your changes

Send relays and internet gateways count on license validation and on server-usage endpoint.
Unifies the funcs for that in the logic package (DRY :] )

## Provide testing steps
Described in ticket

### CODE REVIEW

**Comment by abhishek9686** [comment]
changes should be made in pro pkg look at static failures, these changes will break the pro build

**Review by Aceix — APPROVED** [review]

**Comment by abhishek9686** [comment]
license validation should use the usage func in the pro package, do not move the usage func outside pro

File: pro/util.go [hunk 1]
```diff
@@ -52,5 +51,13 @@ func getCurrentServerUsage() (limits Usage) {
 	if err == nil {
 		limits.Egresses = len(egresses)
 	}
+	relays, err := logic.GetRelays()
```
Comment (abhishek9686) [inline pro/util.go:54]: move GetRelays to Pro pkg

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: pro/util.go (hunk 1 above)
Comment (gabrielseibel1) [reply pro/util.go:54]: https://github.com/gravitl/netmaker/pull/2583/commits/682059c9482e5e7f9fa56c1e4f5c590cb2b8ac4e

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**